streaming/Dockerfile

```diff
@@ -24,9 +24,6 @@ ARG GID="991"
 
 # Apply Mastodon build options based on options above
 ENV \
-# Apply Mastodon version information
```

Comment:
I would rather keep those and add the arguments

Reply:
I agree, and I had added them in originally but then removed it after some thought that it didn't apply to anything in the Streaming code, but better to set them and have them for use them later.